paddlets/analysis/time_domain.py

```diff
@@ -36,10 +36,10 @@ class Seasonality(Analyzer):
     """
     def __init__(self, 
                  period: Optional[int] = None,
-                 nlags: int = 300,
-                 alpha: float = 0.05,
-                 mode: str = 'additive',
-                 order: int = 1,
+                 nlags: Optional[int] = 300,
+                 alpha: Optional[float] = 0.05,
+                 mode: Optional[str] = 'additive',
+                 order: Optional[int] = 1,
```

Comment:
Line39 - Line42 should Not be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional